peachjam/views/legislation.py

```diff
@@ -197,7 +197,7 @@ def get_timeline_events(self):
                     "event": "amendment",
                     "amending_title": amendment.get("amending_title"),
                     "amending_uri": amendment.get("amending_uri"),
-                    "flag": True
+                    "unapplied_amendment": True
                     if amendment.get("date") not in point_in_time_dates
```

Comment:
oh, sorry, this can probably be simplified a bit: 
`"unapplied_amendment": bool(amendment.get("date") not in point_in_time_dates)`?